BasicAPI6/LinkedTables.java

```diff
@@ -0,0 +1,61 @@
+package BasicAPI6;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+// класс для работы с таблицами связанными с экземплярами Notebook
+public class LinkedTables {
+    private HashMap<String, Map<Integer, String>> tables; // связь между ключами Notebook и связанными таблицами
```

Comment:
private final

Reply:
If it is declared as final, data cannot be added  (LinkedTables set method).